app/assets/v2/css/profile.css

```diff
@@ -1,3 +1,4 @@
+/* stylelint-disable no-descending-specificity */
```

Comment:
make sense to add this throughout the site as opposed to just this file ? (not sure where this goes in though )

Reply:
yes because that rule is a pain in the ass using tags instead of class names, and for example every rule with img will complain.